rosdep/python.yaml

```diff
@@ -2296,6 +2384,7 @@ python-rosdistro:
 python-rosinstall:
   arch: [python2-rosinstall]
   fedora: [python-rosinstall]
+  gentoo: [dev-util/rosinstall]
```

Comment:
Please change this to `dev-python/rosinstall`. We'll fix that one if it's not working properly.